src/api-utils.ts

```diff
@@ -14,19 +14,23 @@ interface GitRef {
   };
 }
 
+interface ErrorStatus extends Error {
+  status?: number;
```

Comment:
Just a minor comment and please correct me if I'm wrong, but I think we can also remove the question mark here now that the suggested change has been implemented?